.github/CODEOWNERS

```diff
@@ -23,7 +23,7 @@
 
 # Bazel build rules
 *.bzl               @cfrantz @drewmacrae
-BUILD               @cfrantz @drewmacrae
```

Comment:
Because this line item didn't have a preceding slash I believe it used to point to all files named BUILD in the source tree, not just /BUILD. Can we keep this line and have the new addition?

```
BUILD               @cfrantz @drewmacrae
BUILD.bazel         @cfrantz @drewmacrae
```